usr/lib/xtable/libxtable.c

```diff
@@ -0,0 +1,202 @@
+/*
+* SPDX-FileCopyrightText: (c) 2024 Mohammad Shehar Yaar Tausif <sheharyaar48@gmail.com>
+* SPDX-License-Identifier: MIT OR GPL-2.0-only
+*/
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <lauxlib.h>
+#include <lua.h>
+#include <lualib.h>
+
+#include <errno.h>
+#include <xtables.h>
+
+#include "luaxtable.h"
+
+#ifndef LUAXTABLE_MODULE
+#error "LUAXTABLE_MODULE is not defined"
+#endif
+
+typedef struct libxtable_flags_s {
+	const char *name;
+	lua_Integer value;
+} libxtable_flags_t;
+
+static lua_State *L = NULL;
+const char *libxtable_match_key = "libxtable_match";
+const char *libxtable_target_key = "libxtable_target";
```

Comment:
We don't need this indirection here.. you can just use the actual strings or even better.. use the function pointer.. e.g., `libxtable_match`